101/dotnet/src/101chart/values.yaml

```diff
@@ -0,0 +1,17 @@
+# Default values for 101chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: docker.io
+  pullPolicy: IfNotPresent
```

Comment:
Maybe "always" is more appropriate for dev environment. Also we might have different overrides for dev vs. prod

Reply:
Change to always for now. We can either leave it blank or fill in Prod value afterwards.